backend/src/govsg/middlewares/govsg-verification.middleware.ts

```diff
@@ -65,14 +65,23 @@ export const listMessages = async (
       name: recipient_name,
       mobile: recipient,
       data: JSON.stringify(remainingParams),
-      passcode: row.govsgVerification.passcode,
+      passcode: row.govsgVerification?.passcode,
       sent: row.sentAt,
       officer: officer_name,
     }
   })
+  if (count > 0) {
+    const row = rows[0]
+    return res.json({
+      messages,
+      total_count: count,
+      has_passcode: !!row.govsgVerification?.passcode,
+    })
+  }
   return res.json({
     messages,
     total_count: count,
```

Comment:
In this case, `messages` is an empty array and `count` is 0? Would it be clearer to explicitly state the values and add a comment explaining the case?

Reply:
Yep. Got it, let me refactor this...

Reply:
Done!

Reply:
Did you make the change? 🤔

Reply:
I refactored and that block is not needed anymore. See 2nd latest commit.

Reply:
My bad, thanks for making the change.